libs/datamodel/core/src/transform/ast_to_dml/db/types.rs

```diff
@@ -109,16 +115,36 @@ impl RelationField<'_> {
     }
 }
 
+/// Information gathered from validating attributes on a model.
 #[derive(Default, Debug)]
 pub(crate) struct ModelData<'ast> {
+    /// @(@)id
     pub(crate) primary_key: Option<PrimaryKeyData<'ast>>,
+    /// @@ignore
     pub(crate) is_ignored: bool,
-    /// @(@) index and @(@)unique.
+    /// @@index and @(@)unique.
     pub(crate) indexes: Vec<IndexData<'ast>>,
     /// @@map
     pub(super) mapped_name: Option<&'ast str>,
 }
 
+impl ModelData<'_> {
```

Comment:
@do4gr I didn't go through with it because I don't want to cause gratuitous merge conflicts, but I was just thinking: all these `ModelData`, `EnumData`, etc. structs are _only_ about attributes, and `ModelAttributes` would be a much better name. What do you think?

Reply:
Yes, sounds good. I was also confused when I was looking for the model name in there. If it was called XAttributes it would be clearer